packages/cf-style-container/src/createComponent.js

```diff
@@ -0,0 +1,72 @@
+import { extractPassThroughProps, resolvePassThrough } from 'fela-utils';
+import { createElement } from 'react';
+import PropTypes from 'prop-types';
+
+function createComponentFactory(createElement, contextTypes) {
+  return function createComponent(rule, type, passThroughProps) {
+    const displayName = rule.name ? rule.name : 'FelaComponent';
```

Comment:
Fixed. It's falsy if you use anonymous function as

```js
const Foo = createComponent(() => { my_rule}, 'h1')
```

vs 

```js
const FooStyle = () => { my_rule};
const Foo = createComponent(FooStyle, 'h1')
```

It's like other javascripts. So far we are mostly using that first shorter syntax and doesn't really cause any troubles with debugging I think.